src/GitLinkTask/LinkPdbToGitRemote.cs

```diff
@@ -45,12 +48,20 @@ public override bool Execute()
                 CommitId = GitCommitId,
                 GitWorkingDirectory = GitWorkingDirectory,
                 IndexAllDepotFiles = IndexAllDepotFiles,
+                IntermediateOutputPath = Path.GetFullPath(AddTrailingSlash(IntermediateOutputPath)),
             };
             bool success = Linker.Link(PdbFile.GetMetadata("FullPath"), options);
 
             return success && !Log.HasLoggedErrors;
         }
 
+        private static string AddTrailingSlash(string path)
+        {
+            return path.EndsWith(Path.DirectorySeparatorChar.ToString())
```

Comment:
Technically we could optimize this method by caching the ToString call, but I guess that is premature optimization.

Reply:
We also could just do a last char comparison and avoid ToString altogether. But ya, I was going for simple here since we do it just once.